license-scanner/scanner.ts

```diff
@@ -42,8 +41,7 @@ const scanCrates = async function (rust: ScanOptionsRust, options: Omit<ScanOpti
       .catch(reject);
   });
 
-  if (project.license !== null && project.license !== undefined) {
-    assert(typeof project.license === "string");
+  if (project.license !== null && project.license !== undefined && typeof project.license === "string") {
```

Comment:
if `typeof project.license` is `"string"`, it couldn't be `null` or `undefined`, right?

However, what drove this change at all? Do we have non-string licenses? 
I understand the logic as it was before: it should be impossible to have an existing license that's not a string, and if it happens, throw an error, indicating that's something seriously wrong. 
But now we want to let it pass?

Reply:
Ah right, this is a leftover from my previous changes with `Inheritable<>`, which I have rolled back.
Rolled back this as well - now it's like before.

Thanks for attentiveness 👍